mobile/ios/Podfile

```diff
@@ -33,3 +33,6 @@ target 'CompanionKit-tvOS' do
     # Pods for testing
   end
 end
+
+pod 'Firebase/Auth'
```

Comment:
@rlnsy Clement and I figured out what's going on with the auth pod! Per my previous comment, the firebase functionality is covered with the @react-native-firebase/app package that should be in the ./mobile/package.json file. The issue is that that dependency is not in the code so that's why it's not building and you need the Auth pod. Vitor had told us to add this package in a previous meeting and I must have missed adding that instruction in notion when you were setting up the project - my bad! If you add the react-native-firebase/app dependency and take out the auth pod, and rerun yarn all, it should work. Clement has tested this.